dropwizard-auth/src/test/java/io/dropwizard/auth/basic/BasicAuthProviderTest.java

```diff
@@ -8,8 +8,13 @@
 import java.security.Principal;
 
 public class BasicAuthProviderTest extends AuthBaseTest<BasicAuthProviderTest.BasicAuthTestResourceConfig> {
-    public static class BasicAuthTestResourceConfig extends AuthBaseResourceConfig{
-        protected ContainerRequestFilter getAuthFilter() {
+    public static class BasicAuthTestResourceConfig extends AbstractAuthResourceConfig {
+        public BasicAuthTestResourceConfig() {
+            super();
```

Comment:
I believe, `super()` is redundant, because the Java compiler automatically inserts a call to the default constructor of a superclass.